docs/notebooks/flax_basics.ipynb

```diff
@@ -0,0 +1,911 @@
+{
```

Comment:
<p>(Additionally to what I said in above comment)</p><p> </p><p>Nit : "not column" -&gt; "not column vectors".</p><p> </p><p>Mentioning <code>kernel_init</code> and <code>bias_init</code> here without using it is a bit confusing. I would either remove it or add it to the code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/flax/pull/542/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Good point about the init functions, will remove I guess (otherwise would make the code more complicated than it need).</p><p> </p><p>Added the missing word.</p>